admin/style.css

```diff
@@ -3,7 +3,7 @@ body {
     flex-direction: column;
     justify-content: center;
     align-items: center;
-    background-image: url('/assets/background6.jpeg');
+    background-image: url('./assets/background6.jpeg');
```

Comment:
It says in github pages console that this asset isn't loading (404). This is the culprit but I'm curious the pathing, it might be a ../ thing, I'd need to see it to follow it. I'm a bit visual sometimes when it comes to pathing.